src/main/java/net/sf/jabref/exporter/SaveDatabaseAction.java

```diff
@@ -288,7 +288,10 @@ private boolean saveDatabase(File file, boolean selectedOnly, String encoding) t
             String tryDiff = Localization.lang("Try different encoding");
             int answer = JOptionPane.showOptionDialog(frame, builder.getPanel(), Localization.lang("Save database"),
                     JOptionPane.YES_NO_CANCEL_OPTION, JOptionPane.WARNING_MESSAGE, null,
-                    new String[] {Localization.lang("Save"), tryDiff, Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:off
+                    new String[] {Localization.lang("Save"), tryDiff,
+                            Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:on
```

Comment:
Yes, that is one of the main causes for false positive obsolete translation strings. In @simonharrer's example, "Cancel" will not be found (which I guess is OK in the specific case, but in general there were ten or so false positives removed).